services/document-manager/backend/app/docman/resources/admin.py

```diff
@@ -0,0 +1,119 @@
+import numpy
+import uuid
+
+from werkzeug.exceptions import Forbidden
+from flask import current_app
+from flask_restplus import Resource, reqparse
+from celery import chord
+
+from app.extensions import api
+from app.docman.models.document import Document
+from app.utils.access_decorators import requires_any_of, MINE_ADMIN
+from app.services.object_store_storage_service import ObjectStoreStorageService
+from app.config import Config
+
+
+@api.route('/admin/transfer-docs-to-object-store', doc=False)
+class TransferDocsToObjectStore(Resource):
```

Comment:
This should be implemented as a click command rather than as an API endpoint. See the implementation of the ETLs for example.

Reply:
This file should be removed